### Box.msg

float64 center_x
float64 center_y
float64 center_z

float64 length
float64 width
float64 height

float64 heading

